NHSN/src/test/java/com/nhsn/pages/ReviewPage.java

```diff
@@ -54,6 +59,36 @@ public void clickOnGenerateButton()
         System.out.println("Clicked on Generate Button");
     }
 
+    public void enterSubmitDateOnReviewPage(String date) throws InterruptedException {
+
+        WebElement submitDateTextbox = driver.findElement(By.xpath("//input[@name='dp']"));
```

Comment:
FYI, I think this could be simplified to css with `By.css("input#dp")`

Reply:
I am not strong enough in CSS. Thats why I prefer to use XPath. Please let me know if you have  prefer CSS Selector.